pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  # This mimics ../../../development/interpreters/python/wrap.sh
+  postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
```

Comment:
Well, it is because blueman calls xdg-utils and dhcp with `system()` invocation.
I admit that it is hacky, but wrapping a third time the same executable looks even worse.